README.md

```diff
@@ -153,7 +153,7 @@ the above dependencies are already available as JBoss modules, and should not be
             <dependency>
                 <groupId>org.jberet</groupId>
                 <artifactId>jberet-se-bom</artifactId>
-                <version>2.0.2.Final-SNAPSHOT</version>
+                <version>2.0.4.Final-SNAPSHOT</version>
```

Comment:
the current version is 2.1.2.Final-SNAPSHOT. Can you update?

Reply:
> the current version is 2.1.2.Final-SNAPSHOT. Can you update?

@chengfang Changes updated. Thanks.